src/components/career-form/career-file-upload-type.tsx

```diff
@@ -29,40 +40,159 @@ export const CareerFileUploadType = ({
 }: CareerFileUploadTypeProps) => {
   const fileName = file.name.split('.');
   const name = `category_select.${fileName[0]}`;
-  const state = watch(name);
+  const state = watch(name, 'default');
+
+  useEffect(() => {
+    register(name, {
+      validate: (value) => {
+        return Boolean(value) && value !== 'default';
+      },
+    });
+    setValue(name, 'default', {
+      shouldDirty: true,
+    });
+  }, []);
 
   const onChange = (event) => {
-    setValue(name, event.target.value);
+    setValue(name, event, {
+      shouldDirty: true,
+    });
     clearError(name);
   };
 
   return (
-    <div>
-      <StyledSelect
+    <SelectWrapper>
+      <StyledListboxInput
         id={name}
-        {...register(name, {
-          validate: (value) => value !== 'default',
-        })}
+        aria-labelledby={name}
         value={state || 'default'}
         onChange={onChange}
       >
-        <option value={'default'} disabled>
-          File category
-        </option>
-        <option value="cv">CV</option>
-        <option value="cover-letter">Cover-Letter</option>
-        <option value="other">other</option>
-      </StyledSelect>
+        <ListboxButton arrow={<SelectArrow />} />
+        <ListboxPopover portal={false}>
+          <ListboxList>
+            <ListboxOption value={'default'} disabled>
+              File category
+            </ListboxOption>
+            <ListboxOption value="cv">CV</ListboxOption>
+            <ListboxOption value="cover-letter">Cover-Letter</ListboxOption>
+            <ListboxOption value="other">other</ListboxOption>
+          </ListboxList>
+        </ListboxPopover>
+      </StyledListboxInput>
       {errors.category_select?.[fileName[0]] && (
-        <FormError
-          error={{ message: 'Deine Auswahl fehlt' }}
-          lineHeight={100}
-        />
+        <FormErrorWrapper>
+          <FormError
+            error={{ message: 'Deine Auswahl fehlt' }}
+            lineHeight={90}
```

Comment:
We are drifting into Denglish with our mix of German and English 😅 This is not a problem here but I'm looking forward to our proper internationalization support